axum/src/handler/mod.rs

```diff
@@ -30,6 +30,9 @@
 //!         Err(StatusCode::BAD_REQUEST)
 //!     }
 //! }
+//!
+//! // See "Error handling" section of the documentation for more examples of
+//! // of Result-returning handlers.
```

Comment:
I think we should just link to the two examples again here, instead of making people go find it themselves. People might also get confused about "service level errors" which might be relevant to them yet.

Reply:
Copied the text, now it is inline in handler docs.